utils/config.go

```diff
@@ -42,6 +41,11 @@ func GetConfig(pathToFile string) (schemas.Config, error) {
 		fieldValue := dataValue.Field(i).Interface()
 		fieldName := typeOfData.Field(i).Name
 
+		// STAKER is optional
+		if fieldName == "STAKER" {
+			continue
+		}
+
 		if fieldValue == "" {
```

Comment:
There are a couple configs vars that we can omit if we are a staker, in that case, we shouldnt bother to validate to see if the configs are missing for those variables